tests/admin/scan/scan.test.ts

```diff
@@ -103,8 +105,14 @@ describe('POST /admin/scan/:qrcode', () => {
       .expect(500, { error: Error.InternalServerError });
   });
 
-  it('should scan the ticket', () =>
-    request(app).post('/admin/scan').send(validBody).set('Authorization', `Bearer ${adminToken}`).expect(204));
+  it('should scan the ticket and return the custom message', async () => {
```

Comment:
Vu qu'on a changé de direction en cours de route, faudrait remettre ça:
```suggestion
  it('should scan the ticket and return the updated user', async () => {
```
A voir si t'as la foi de tester les autres propriétés de l'objet *(en vrai ça pourrait être pas mal)*

Reply:
Peut être pas toutes les propriétés mais vérifier aussi au moins que le `scannedAt` ne soit pas nul.